scripts/appleseed.package.py

```diff
@@ -786,17 +807,17 @@ def main():
     print("")
 
     settings = Settings()
-    package_info = PackageInfo(settings)
+    package_info = PackageInfo(settings, no_zip)
 
     settings.load()
     package_info.load()
 
     if os.name == "nt":
-        package_builder = WindowsPackageBuilder(settings, package_info)
+        package_builder = WindowsPackageBuilder(settings, package_info, no_zip)
```

Comment:
Do we really need to pass `no_zip` to all package builders if it's already available via `PackageInfo`?